common/app/implicits/FaciaContentFrontendHelpers.scala

```diff
@@ -28,23 +28,16 @@ object FaciaContentFrontendHelpers {
     def mainYouTubeMediaAtom: Option[MediaAtom] =
       for {
        main <- faciaContent.properties.maybeContent.map(_.fields.main)
-       atoms <-  faciaContent.properties.maybeContent.flatMap(_.atoms)
+       mediaAtoms <-  faciaContent.properties.maybeContent.map(_.elements.mediaAtoms)
        document <- Some(Jsoup.parse(main))
        atomContainer <- Option(document.getElementsByClass("element-atom").first())
        bodyElement <- Some(atomContainer.getElementsByTag("gu-atom"))
        atomId <- Some(bodyElement.attr("data-atom-id"))
-       mainMediaAtom <- atoms.media.find(ma => (ma.id == atomId && !ma.expired.getOrElse(false)) && ma.assets.exists(_.platform == MediaAssetPlatform.Youtube))
+       mainMediaAtom <- mediaAtoms.find(ma => (ma.id == atomId && !ma.expired.getOrElse(false)) && ma.assets.exists(_.platform == MediaAssetPlatform.Youtube))
```

Comment:
I know not your code but this is a little hard to read (and there are unnecessary extra braces). Maybe a sign we should pull out this predicate `def isMainMediaAtom`.